translation/dest/site/sk-SK.xml

```diff
@@ -330,7 +330,7 @@
     <item quantity="one">Potrebuješ odohrať ešte %1$s %2$s hodnotenú partiu</item>
```

Comment:
I don't know why, but on crowdin is correct in all forms:
`Potrebujete odohrať ešte ...`

but on Github aj see `Potrebuješ odohrať ešte ...` but it's not on crowdin...